exercises/spiral-matrix/src/main/java/SpiralMatrixBuilder.java

```diff
@@ -0,0 +1,5 @@
+class SpiralMatrixBuilder {
+
+
+
+}
```

Comment:
No, according to the policy after the first 20 exercises no starter implementation needs to be provided. But quite a few exercises after the first 20 have an empty starter implementation like this. Do you think that's okay or should we make them more consistent?